pages/common/usql.md

```diff
@@ -0,0 +1,36 @@
+# usql
+
+> Universal CLI interface for SQL databases.
+> More information: <https://github.com/xo/usql>.
+
+- Connect to a specific database with SSL disabled:
+
+`usql {{sqlserver|mysql|postgres|sqlite3|...}}://{{username}}:{{password}}@{{host}}:{{port}}/{{database_name}}?sslmode=disable`
```

Comment:
```suggestion
`usql {{sqlserver|mysql|postgres|sqlite3|...}}://{{username}}:{{password}}@{{host}}:{{port}}/{{database_name}}?sslmode=disable`
```

Why have only an example where SSL is disabled?

Reply:
Well, it's a way to show an additional feature without adding one more example, but maybe it's better to put it with SSL enabled, so I'll remove this part.